ux.symfony.com/public/site.webmanifest

```diff
@@ -1 +1,350 @@
-{"name":"Symfony UX","short_name":"Symfony UX","icons":[{"src":"/android-chrome-192x192.png","sizes":"192x192","type":"image/png"},{"src":"/android-chrome-512x512.png","sizes":"512x512","type":"image/png"}],"theme_color":"#ffffff","background_color":"#ffffff","display":"standalone"}
+{
```

Comment:
Was this file generated or written by hand?

Reply:
It is generated by the command provided by the bundle.

Reply:
Should it be committed or running that command be part of the deploy process?

Could it be integrated (in theory) with AssetMapper so that this file is generated dynamically through that? The URL would be something like`/assets/site-{HASH}.webmainfest`, I think. I don't think that's a problem - but I'm not positive. If we did this, the file would be available in dev automatically and would be naturally dumped during deploy with `asset-map:compile`.

Reply:
Until now I saw the web manifest as a static file generated by the bundle. But indeed it is interesting to leverage on Asset Mapper for having it during dev and ease the deployment.
This could be a way to answer your question on the meta tag `theme-color`, because this value must be present in the manifest and `head` to be used.

The main question to me is: how to manage the assets? Like asset Mapper, the bundle generates the links to the icons and screenshots, but also manipulates them to have them all in the specified format and sizes.
For example with an "app.svg" icon, the bundle may have to create 3 different files depending on the requested sizes and in another format (96x96, 256x256 and 512x512 in PNG format). On this point, I don't know how Asset Mapper can work.

But it is definitively a very good idea to leverage on it => https://github.com/Spomky-Labs/phpwa/issues/28